src/components/CaptureMatching/CurrentCaptureNumber.js

```diff
@@ -1,32 +1,69 @@
 import React from 'react';
-
+import { makeStyles } from '@material-ui/core/styles';
 import { Paper, Typography, Box } from '@material-ui/core';
 
+const useStyles = makeStyles((t) => ({
+  test: {
+    padding: t.spacing(1),
+  },
+  box1: {
+    width: 173,
+    height: 50,
+  },
+  box2: {
+    display: 'flex',
+    justifyContent: 'flex-start',
+    alignItems: 'center',
+    height: '100%',
+    width: '100%',
+  },
+  box3: {
+    //padding: t.spacing(2),
+    width: 48,
+    justifyContent: 'center',
+    display: 'flex',
+    '& svg': {
+      width: 24,
+      height: 24,
+      fill: 'gray',
+    },
+  },
+  box4: {},
+  text: {
+    fontFamily: 'Roboto',
```

Comment:
Is there a reason for overriding the default font family here (Lato, Roboto, Helvetica, Arial, sans-serif)?